src/OSPSuite.UI/Views/HeavyWorkView.cs

```diff
@@ -34,15 +32,21 @@ public void AttachPresenter(IHeavyWorkPresenter presenter)
 
       private void SetLayout()
       {
+         ShowInTaskbar = false;
          if (CancelVisible)
          {
             FormBorderStyle = FormBorderStyle.FixedToolWindow;
             MaximizeBox = false;
-            MaximizeBox = false;
-            ShowInTaskbar = false;
             this.Height = this.MaximumSize.Height;
          }
-            
+         else
+         {
+            this.FormBorderStyle = FormBorderStyle.None;
+            //set the transparencyKey to a color different from the one used on the controls
+            this.BackColor = Color.Black;
+            this.TransparencyKey = Color.Black;
```

Comment:
Would this mean that black text also appears transparent for example? And if so, does it make sense to just 

`this.TransparencyKey = this.BackColor`

Reply:
There are many skins that user could pick, so I wonder about the black color